content/docs/12-enterprise-version/Helm Chart Install References

```diff
@@ -0,0 +1,161 @@
+---
+title: "Helm Chart Install References"
+metaTitle: "Helm Chart Install References"
+metaDescription: "Reference page for Pallete Helm chart self-installation values.yaml file parameters."
+icon: ""
+hideToC: true
+fullWidth: true
+---
+
+# Helm Chart Install Reference Guide
+You can install the Palette Helm Chart in a multi-node Kubernetes cluster for use in production environments. The [Self-hosted Palette installation using Helm](https://docs.spectrocloud.com/enterprise-version/deploying-palette-with-helm) allows users to customize as per their needs by specifying user-defined values in the **values.yaml file.**
+This reference guide explains the parameters available in the **values.yaml** file from the Helm Chart for your installation.
+
+## Mandatory parameters
+
+The following parameters are mandatory for the self-hosted Palette installation using Helm. 
```

Comment:
```suggestion
The following parameters are mandatory. 
```
We mentioned in the previous paragraph that this is for Palette self-hosted install. So we already established context.